boards/nuvoton/npcx4m8f_evb/doc/index.rst

```diff
@@ -62,7 +62,7 @@ The following features are supported:
 Other hardware features are not currently supported by Zephyr (at the moment)
 
 The default configuration can be found in the defconfig file:
-``boards/arm/npcx4m8f_evb/npcx4m8f_evb_defconfig``
+``boards/nuvoton/npcx4m8f_evb/npcx4m8f_evb_defconfig``
```

Comment:
can you update to `zephyr_file` where not used now that you are at it?

Reply:
yeah good idea